istio-control/istio-autoinject/files/injection-template.yaml

```diff
@@ -11,10 +11,9 @@ template: |
   {{- else }}
     image: "{{ .Values.global.hub }}/{{ .Values.global.proxy_init.image }}:{{ .Values.global.tag }}"
   {{- end }}
-    command:
-    - "/usr/local/bin/istio-iptables.sh"
+    args:
```

Comment:
It's intentional - command is better, allows specifying the script. We may run proxy image as proxy_init to improve startup.